lib/view/license_view.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+
+import '../strings.dart';
+import '../style.dart';
+
+class LicenseView extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    // Using showLicensePage directly gives the wrong style for app bar, and
```

Comment:
@rockgecko-development is this a bug in the license widget we should report to Flutter SDK repo?

Reply:
Yeah happy with this until we can investigate the showLicensePage issues. 👍

Reply:
I think it's caused by the default theme that's configured. It was a green titlebar... I think after merging @workerbee22 's PR last night it became white text on white bg. Not sure if a bug in LicensePage, or a problem with how we have set the global app theme.
Either way, it's not what we want and I didn't see any other way around